src/10-webpage-publishing-with-ipfs/index.md

```diff
@@ -0,0 +1,308 @@
+author: David Dias
+collection: posts
+---
+
+> - Don't you hate it when your links break?
+> - Don't you hate it when your web host gets attacked and your websites go down?
+> - Don't you hate managing servers just to self-host a web page?
+> - Don't you hate it when you switch cloud providers and all the data links change?
+> - Don't you hate it when people relying on your content get broken embeds?
+
+> NOTE: I think it'd be better to start stronger than rhetorical questions. For instance:    
+> The web is broken. Links break; web hosts get attacked, and websites go down (often permanently). As a web developer, you have to by a sysadmin just to host a web page. If you switch cloud providers, links break. Even with a fully functional website, more issues might pop up; link rot, or broken embeds for third parties serving your content. It's 2016, and not all is as it should be.
+
+Publishing web content has become a very complex, confusing, and precarious process. Publishing static content -- simple web pages, images, videos, or documents -- is way harder than it should be. The "easy way" means depending on services that may change things under you or shut down completely. Hundreds of millions of people have been served, sometimes overnight, with a "Sorry, we're going in a different direction" message. The lucky ones get a dump of their data and a short timeline to "get their affairs in order" and rehost their content. This means hard work rebuilding the site elsewhere, and -- for those who were not extremely careful with domain names and URLs up-front -- attempting to update soon-to-be-broken links all over the web. Like changing mailing addresses after moving, most resign themselves to suffering some things breaking forever.
+
+The alternative -- self-hosting your content on infrastructure you control -- is not much better. Only the zealous few manage their own hardware today. And those who compromise by hosting their own websites on virtual private servers run into a myriad of complex and brittle processes. Even very experienced web developers can struggle with the complexities of setting up servers, replicating across regions, adding load-balancers, securing machines against hackers, responding to sudden problems, and just the constant grind of long-term maintenance. All of this requires large amounts of domain specific knowledge that newcomers find very difficult to use.
+
+"I just wanted to publish my personal website!" It is not a mystery why most people use Facebook, Twitter, Medium, and Tumblr to publish and host their public personas, their blogs, and all their thoughts.
+
+Suggestion: This is where IPFS, the permanent web, comes in. We've been working on a solution to these issues, imagining a word where it is easier to publish web content; where it easier to host and maintain it; where links don't break or change; where old versions didn't disappear in moment; and where all of this is as simple as tweeting. All at once.
```

Comment:
Same with this paragraph. I think it is stronger than the questions.

Again, I think these questions come off as a bit weak. People skim through repetitive questions, only reading the right edge and not getting the rhetorical effect that you'd get in speech. Rephrase, like above.